components/Statistics/__tests__/IconStat-test.tsx

```diff
@@ -0,0 +1,74 @@
+    expect(view).toMatchSnapshot();
+  });
+  it("should render the icon and font in correct size", () => {
+    const view = render(
+      <IconStat icon="home" iconSize={30} fontSize={21}>
+        1001km
+      </IconStat>,
+    );
+    expect(view).toMatchSnapshot();
+  });
+  describe("should render the correct icon color depending on the given status", () => {
+    it("primary", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="primary" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-primary-500"]);
+    });
+    it("basic", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="basic" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-basic-500"]);
+    });
+    it("success", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="success" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-success-500"]);
+    });
+    it("info", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="info" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-info-500"]);
+    });
+    it("warning", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="warning" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-warning-500"]);
+    });
+    it("danger", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="danger" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-danger-500"]);
+    });
+    it("text", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="text" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofLightTheme["text-basic-color"]); // wäre im Darkmode das andere Theme
+    });
+    it("incorrect syntax should render primary color", () => {
+      const { getByTestId } = render(<IconStat icon="home" status="xyz" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-primary-500"]);
+    });
+    it("no given status should render primary color", () => {
+      const { getByTestId } = render(<IconStat icon="home" />);
+      const view = getByTestId("container");
+      const icon = view.props.children[0];
+      expect(icon.props.style.color).toBe(foofTheme["color-primary-500"]);
+    });
```

Comment:
Hier vielleicht lieber das Keyword `test` statt `it` nehemen da es sich beser liest. Die beiden machen aber exakt das gleiche. Oder du änderst die Testbeschreibung